p2p/src/main/java/bisq/network/p2p/network/NetworkNode.java

```diff
@@ -98,7 +98,8 @@
 
     public SettableFuture<Connection> sendMessage(@NotNull NodeAddress peersNodeAddress,
                                                   NetworkEnvelope networkEnvelope) {
-        log.debug("sendMessage: peersNodeAddress=" + peersNodeAddress + "\n\tmessage=" + Utilities.toTruncatedString(networkEnvelope));
+        if (log.isDebugEnabled())
+            log.debug("sendMessage: peersNodeAddress=" + peersNodeAddress + "\n\tmessage=" + Utilities.toTruncatedString(networkEnvelope));
```

Comment:
We should use the {} syntax instead of +

Reply:
good point but it has no perfomance implications in that case. maybe + is even faster if we already checked for `isDebugEnabled`?